.travis.yml

```diff
@@ -7,12 +7,12 @@ env:
   matrix:
     # Each line is a set of environment variables set before a build.
     # Thus each line represents a different build configuration.
-    - SHADERC_BUILD_TYPE=Release SHADERC_CODE_COVERAGE=OFF
-    - SHADERC_BUILD_TYPE=Debug SHADERC_CODE_COVERAGE=OFF
+    #- SHADERC_BUILD_TYPE=Release SHADERC_CODE_COVERAGE=OFF
```

Comment:
Should add TODO saying you expect to turn this on soon.
Otherwise, there is a question as to why this would have started failing.  Did it fail?